src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java

```diff
@@ -161,22 +171,14 @@ private TenantTable getTenantTableFromHashKey(String hashKey) {
     }
 
     private String getTableDescriptionTableName() {
-        try {
-            cache.get(tableDescriptionTableName, () -> {
-                createTableDescriptionTableIfNotExists(pollIntervalSeconds);
-                return new TableDescription().withTableName(tableDescriptionTableName);
-            });
-        } catch (ExecutionException e) {
-            throw new RuntimeException(e);
-        }
-        return tableDescriptionTableName;
+        return tableDescriptionSupplier.get().getTableName();
```

Comment:
brought this up in slack last week, I found it weird that we were caching this global table name per tenant context, I think to lazy create the metadata table using an existing tried and tested pattern. Ran into this on the empty virtual list table test case, given the scan tried running against a non-existant table. rather than forcing the table creation for that test, I tried cleaning this up as promised.